### DISCUSSION TITLE
There is a large amount of network traffic between etcd clusters

### DISCUSSION THREAD

**Discussion by Rmond** [etcd-io/etcd#18552] [Q&A]
![微信截图_20240906110316](https://github.com/user-attachments/assets/ff9049b0-6815-47fa-8000-3d80ba289d62)
my etcd version is 3.4.3 and I suddenly realized that the network traffic between clusters is very large.
![微信截图_20240906110644](https://github.com/user-attachments/assets/1b153ad6-d805-4e99-b03d-0d34f4ab6569)

**Comment by jmhbnz** [comment]
Hi @Rmond - Thanks for raising this discussion, network traffic between etcd peers is expected.  If you have concerns on the level of network traffic please provide more detail such as:
- Full etcd logs
- Network metrics https://etcd.io/docs/v3.4/metrics/#network
- Details on the size of your etcd database and how it is being used
